### PR TITLE
Update Oracle Go SDK

### PR DESCRIPTION
**Pull request by trown**
Currently, the oracle-oci builder is broken, because it is unable
to read the key file provided by Oracle cloud. Updating to the
latest compatible oci-go-sdk fixes the issue.

### CODE REVIEW

**Review by sylviamoss — COMMENTED** [review]
That's a huge bump! You haven't bumped to the latest available version because of breaking changes, right? 
This seems to be ok, but I don't have the environment to run a test so better wait for the codeowner to give their 👍🏼 .

**Comment by trown** [comment]
Ya I bumped to the latest version I could without breaking changes. The builder is totally broken with the current API keys that Oracle gives out though without this bump.

With this, it is at least working for me with a simple shell provisioner.

**Review by azr — APPROVED** [review]
Sounds like a good reason to merge if things didn't work before this one. Though I don't have an access either.

Maybe let's wait to have another green light for this one.

**Comment by azr** [comment]
Pinging @prydie @owainlewis, as you are codeowners of the oracle part: 

https://github.com/hashicorp/packer/blob/f578b93f7ecbf6d08aae52a932294739abb6157b/CODEOWNERS#L37-L38

Does this look okay to you ?

**Comment by trown** [comment]
terraform apply staging

**Comment by SwampDragons** [comment]
It's been a bit with no response, so I'm gonna merge this and if the oracle codeowners weigh in if they have any concerns. We're in the beginning of a release cycle so we have time to revert if there are issues.

**Pull request merged**
